python/python/lance/dataset.py

```diff
@@ -288,6 +288,31 @@ def versions(self):
             v["timestamp"] = datetime.fromtimestamp(v["timestamp"])
         return versions
 
+    def create_index(
+        self, column: str, index_type: str, name: Optional[str] = None, **kwargs
+    ):
+        """Create index on column
+
+        *Experimental*
+
+        Parameters
+        ----------
+        column : str
+            The column to be indexed.
+        index_type : str
+            The type of the index. Only support "IVF_PQ" now.
+        name : str, optional
+            The index name. If not provided, an index name will be generated from the
+            column name.
+        kwargs :
```

Comment:
can you explicitly document the num partitions and num subvectors parameters?
also 1) indicate what the default values are and 2) add a link to those papers maybe?

Reply:
Ok, will do.